chapters/data-analysis.tex

```diff
@@ -1,182 +1,236 @@
+Transforming raw data into a substantial contribution to scientific knowledge
+requires a mix of subject expertise, programming skills,
+and statistical and econometric knowledge.
+The process of data analysis is, therefore,
+a back-and-forth discussion between people
+with differing skill sets.
+The research assistant usually ends up being the pivot of this discussion.
 It is their job to translate the data received from the field into
-economically meaningful indicators and to analyze them 
+economically meaningful indicators and to analyze them
 while making sure that code and outputs do not become too difficult to follow or get lost over time.
+This can be a complex process.
 
-When it comes to code, though, analysis is the easy part, 
-\textit{as long as you have organized your data well}. 
-Of course, there is plenty of complexity behind it: 
+When it comes to code, though, analysis is substantially less intricate than data cleaning,
+as long as you have already organized your data well.
+Of course, there is plenty of complexity behind it:
 the econometrics, the theory of change, the measurement methods, and so much more.
-But none of those are the subject of this book. 
-\textit{Instead, this chapter will focus on how to organize your data work so that coding the analysis becomes easy}.
-Most of a Research Assistant's time is spent cleaning data and getting it into the right format. 
+However, none of those are the subject of this book,
+and most statistical software packages complex functions into simple expressions.
+This chapter will focus on how to organize your data work so that coding the analysis is simple.
+Most of a research assistant's time is spent cleaning data and getting it into the right format.
 When the practices recommended here are adopted,
-analyzing the data is as simple as using a command that is already implemented in a statistical software. 
-
+analyzing the data is as simple as using a command that is already implemented in a statistical software.
 
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Data management}
-The goal of data management is to organize the components of data work 
-so it can traced back and revised without massive effort.
-In our experience, there are four key elements to good data management: 
-folder structure, task breakdown, master scripts, and version control. 
+\section{Managing data effectively}
+
+The goal of data management is to organize the components of data work
+so the complete process can traced, understood, and revised without massive effort.
+In our experience, there are four key elements to good data management:
+folder structure, task breakdown, master scripts, and version control.
 A good folder structure organizes files so that any material can be found when needed.
 It reflects a task breakdown into steps with well-defined inputs, tasks, and outputs.
 This breakdown is applied to code, data sets, and outputs.
```

Comment:
```suggestion
This breakdown is applied to code, data sets, and research results.
```

Output is the best word here, but it is used in the sentence immediately before, and is used in a more general meaning the first time. Can this be improved somehow?

Reply:
"It reflects a breakdown of the data work process into steps with well-defined inputs, tasks, and outputs."?
And then I think we can take the line you changed out. It doesn't really add much at this point.

Reply:
Changed here: 262585957f223ee49d6e72565f8f859bd59316b6